index.markdown

```diff
@@ -1067,6 +1067,34 @@ incurring them.
 We manage expenses using Xero. For more information about how to do this, see
 the [guide on claiming expenses](/guides/claiming-expenses)
 
+#### Travel and Accommodation Policy
+
+Currently, the default for everyone in the organisation is working from home.
+
+For this period, we will consider regular travel to an office space without a business critical need as commuting.
+
+Commuting costs are the responsibility of individuals, while dxw will fund business critical travel and accommodation.
+
+A business critical need is defined as:
+
+* Client meetings
+* Meetings with your line manager (e.g. during probation, monthly check-ins, quarterly reviews, post parental / sick leave)
+* Team meetups (e.g. design community, tech forum, away days, Board meetings)
+* All-staff in-person events
+* Conferences and training events
+
+At present, there is no obligation to attend in-person meetings.
+
+As with all company expenses, please keep in mind that costs should be necessary and proportionate. We rely on everyone to be conscious of the impact of their travel on the environment. If you are able to take public transport, over being driven / driving yourself, please do.
+
+Accommodation will be covered for business critical events that require overnight stays.
+
+[There are guidelines including rates and a list of recommended hotels.](https://docs.google.com/document/d/1HCNkW_wgJ_CDKXGNwdcsx64-F2AZtbfML7_Tb_PvDRA/edit)
+
+Accommodation should be booked by the individual on a Pleo or expensed.
```

Comment:
I suggest 'a Pleo company debit card' or similar to make it clear what a 'Pleo' is (I only learned the other day)